packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,204 @@
+  forceEmail?: string;
+  canGoBack?: boolean;
+};
+
+type FormData = {
+  email: string;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPassword = ({
+  serviceName,
+  forceEmail,
+  canGoBack,
+}: ResetPasswordProps & RouteComponentProps) => {
+  usePageViewEvent('settings.reset-password', {
+    entrypoint_variation: 'react',
+  });
+
+  const [email, setEmail] = useState<string>(forceEmail || '');
+  const [emailErrorText, setEmailErrorText] = useState<string>();
+  const [isFocused, setIsFocused] = useState(false);
+  const alertBar = useAlertBar();
+  const account = useAccount();
+  const { l10n } = useLocalization();
+  const navigate = useNavigate();
+  const pageId: string = 'reset-password';
+  const onFocusMetricsEvent = 'reset-password.engage';
+
+  const { handleSubmit } = useForm<FormData>({
+    mode: 'onBlur',
+    criteriaMode: 'all',
+    defaultValues: {
+      email: '',
+    },
+  });
+
+  const onFocus = () => {
+    if (!isFocused && onFocusMetricsEvent) {
+      logViewEvent(settingsViewName, onFocusMetricsEvent, {
+        entrypoint_variation: 'react',
+      });
+      setIsFocused(true);
+    }
+  };
+
+  const navigateToConfirmPwReset = useCallback(() => {
+    // TODO: verify if passwordForgotToken and clearQueryParams needed
+    navigate(HomePath + 'confirm_reset_password', { replace: true });
+  }, [navigate]);
+
+  const onSubmit = useCallback(
+    async (email) => {
+      try {
+        logViewEvent(settingsViewName, 'reset-password.submit', {
+          entrypoint_variation: 'react',
+        });
+        // TODO password reset method
+        await account.resetPassword(email);
+        logViewEvent(settingsViewName, 'reset-password.success', {
+          entrypoint_variation: 'react',
+        });
+        navigateToConfirmPwReset();
+      } catch (e) {
+        console.log('reset-password.fail');
+        logViewEvent(settingsViewName, 'reset-password.fail', {
+          entrypoint_variation: 'react',
+        });
+        // TODO complete error handling - case 'UNKNOWN_ACCOUNT'
+        // TODO localize error message
+        // TODO pass sign up link to tooltip
+        setEmailErrorText('Unknown account');
+
+        // TODO complete error handling - case 'USER_CANCELED_LOGIN'
+        // logEvent('login.canceled')
+
+        // TODO error handling for other errors
+        alertBar.error(
+          l10n.getString(
+            'reset-password-error',
+            null,
+            'Sorry, there was a problem resetting your password'
+          )
+        );
+      }
+    },
+    [l10n, navigateToConfirmPwReset, alertBar, account]
+  );
+
+  return (
+    <>
+      <HeaderContinueTo
+        prefixDataTestId={pageId}
+        prefixFtlId={pageId}
+        headingText="Reset password"
+        serviceName={serviceName}
+      />
+
+      {/* TODO: Review a11y */}
+      <WarningMessage
+        prefixFtlId={pageId}
+        prefixDataTestId={pageId}
+        warningType="Note:"
```

Comment:
I am not sure what other "warning types" we might have but we could potentially set this to an enum if there's a set number.

I won't call out all the instances btw but we'll again need to not set those l10n IDs dynamically and try to avoid the data-testIDs if we can.

Reply:
I like that idea - but might put that in with #14627